tools/cloud-build/provision/reservations.tf

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+locals {
+  reservation_description = "Reservation for running tests. Managed by Terraform. DO NOT modify manually."
+}
+
+resource "google_compute_reservation" "c2standard60_us_west4_c" {
+  name        = "c2standard60-us-central1-a"
+  zone        = "us-west4-c"
+  description = local.reservation_description
```

Comment:
will this even work? don't we run this with placement groups? I suspect the reservation will not be taken....

Reply:
Some tests are disabling placements. E.g. `community/examples/hpc-slurm-chromedesktop.yaml`